README.md

```diff
@@ -274,6 +254,11 @@ Elevation.lineElevation({
   console.log(str)
 });
 ```
+## Developement Setup
+
+Clone the openrouteservice-js repository from GitHub into a developement environment of your choice.
+
+Install the dependencies with ```npm install```
```

Comment:
You can add the git command here as well:
`git clone ...`
and the 'enter the directory' command.

Use only single ` if you do inline code. When using a code block (3x ` ) use new line. Also see bottom part of :
https://github.com/GIScience/openrouteservice-js/pull/49#discussion_r945556878